deployments/avalanche.diamond.immutable.json

```diff
@@ -1,92 +1,12 @@
 {
   "LiFiDiamondImmutable": {
```

Comment:
What happened here? This doesnt look right...facets are missing. Maybe run script again?